src/elf_message/elf_message_field.c

```diff
@@ -1,33 +1,33 @@
 #include "global.h"
-#include "z64elf_message.h"
+#include "z64hint_commands.h"
 
-ElfMessage gOverworldNaviMsgs[] = {
-    ELF_MSG_FLAG(CHECK, 0x40, false, EVENTCHKINF_05),
-    ELF_MSG_FLAG(CHECK, 0x41, false, EVENTCHKINF_09),
-    ELF_MSG_FLAG(CHECK, 0x42, false, EVENTCHKINF_12),
-    ELF_MSG_FLAG(CHECK, 0x43, false, EVENTCHKINF_14),
-    ELF_MSG_FLAG(CHECK, 0x44, false, EVENTCHKINF_40),
-    ELF_MSG_SONG(CHECK, 0x45, false, ITEM_SONG_SARIA),
-    ELF_MSG_STRENGTH_UPG(CHECK, 0x46, true, 0),
-    ELF_MSG_FLAG(CHECK, 0x47, false, EVENTCHKINF_25),
-    ELF_MSG_MAGIC(CHECK, 0x48, false),
-    ELF_MSG_FLAG(CHECK, 0x49, false, EVENTCHKINF_33),
-    ELF_MSG_FLAG(CHECK, 0x4A, false, EVENTCHKINF_37),
-    ELF_MSG_FLAG(CHECK, 0x4B, false, EVENTCHKINF_80),
-    ELF_MSG_FLAG(CHECK, 0x4C, false, EVENTCHKINF_43),
-    ELF_MSG_FLAG(CHECK, 0x4D, false, EVENTCHKINF_45),
-    ELF_MSG_ITEM(CHECK, 0x4E, true, ITEM_HOOKSHOT, ITEM_NONE),
-    ELF_MSG_MEDALLION(CHECK, 0x50, false, ITEM_MEDALLION_FOREST),
-    ELF_MSG_MEDALLION(CHECK, 0x51, false, ITEM_MEDALLION_FIRE),
-    ELF_MSG_BOOTS(CHECK, 0x52, false, ITEM_BOOTS_IRON),
-    ELF_MSG_MEDALLION(CHECK, 0x53, false, ITEM_MEDALLION_WATER),
-    ELF_MSG_FLAG(CHECK, 0x54, false, EVENTCHKINF_AA),
-    ELF_MSG_ITEM(CHECK, 0x55, true, ITEM_LENS, ITEM_NONE),
-    ELF_MSG_MEDALLION(CHECK, 0x57, false, ITEM_MEDALLION_SHADOW),
-    ELF_MSG_SONG(CHECK, 0x58, false, ITEM_SONG_REQUIEM),
-    ELF_MSG_STRENGTH_UPG(CHECK, 0x56, true, 1),
-    ELF_MSG_MEDALLION(CHECK, 0x5A, false, ITEM_MEDALLION_SPIRIT),
-    ELF_MSG_ITEM(CHECK, 0x5B, true, ITEM_ARROW_LIGHT, ITEM_NONE),
-    ELF_MSG_FLAG(CHECK, 0x5C, false, EVENTCHKINF_C3),
-    ELF_MSG_END(0x5F),
+static HintCmd sOverworldNaviHints[] = {
```

Comment:
I think it makes more sense to keep those global, even tho the symbol isn't used directly, since the data is technically used globally and having something used globally as a static variable feels wrong/confusing

Reply:
Note this data is not in code, it's loaded separately

Reply:
I have no opinion on this matter personally, and will do whatever the consensus is. I inherited this change from dragorns initial PR that I build my changes on top of

Reply:
tho it is important to keep in mind that we treat things in other dma'd files that arent always accessed directly like scenes and objects as global. all of these should prob get the same treatment